coralproject/talk
# Modqueue live new comments

## What does this PR do?
- Implement live updates of new comments in the moderation queues
- Improve keyboard navigation
  - When reaching the end and going to the next comment, it will load new comments if available and select the next new one.
- Improve single view modus
  - Seamless browsing through newer and load more comments without clicking on any buttons.
  - Switching from single view to normal view will automatically scroll to the selected comment
- Refactor
  - The monolith `styles.css` in the Moderation route is finally gone and split up into separate files for each Component 🎉 
  - Added proptypes

## How do I test this PR?

- Add new comments and see a button appear in the appropriate mod queue!
- Clicking on view more comments will load them into the mod queue
- Use keyboard browsing and try to browse past the last comment where it should trigger loading more comments and select the next one coming
- Check out interaction with single view modues, browsing through current and new comments. A loading more button is no longer necessary as it is done automatically now.
